packages/example/src/app/layout.tsx

```diff
@@ -0,0 +1,29 @@
+import {ReactNode} from 'react';
+// import Provider from './Provider';
+import ServerOnlyContext from './ServerOnlyContext';
+
+type Props = {
+  children: ReactNode;
+};
+
+export default function RootLayout({children, ...rest}: Props) {
+  console.log(rest);
+
+  // How to get this from the URL?
```

Comment:
You can get through layout props

https://beta.nextjs.org/docs/routing/defining-routes#example
_*works in layout also_